### PR TITLE
Apply renames to markdown contents

### PR DESCRIPTION
**Pull request by DavidCurtiss**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-python/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Applies recent file renames to the markdown files

### Why should this Pull Request be merged?

Keep the markdown files in sync

### What testing has been done?

None

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
-# Contributing to Measurement Plugin Support for Python
+# Contributing to Measurement Plug-In Support for Python
```
Comment (dixonjoel) [inline CONTRIBUTING.md:1]: Similar to LabVIEW, we'd like to call the repo 'title' `Measurement Plug-In SDK for Python`.

**Review by dixonjoel — CHANGES_REQUESTED** [review]
Will take another quick look after the Measurement Plug-In SDK for Python phrase is updated.

File: CONTRIBUTING.md (hunk 1 above)
Comment (DavidCurtiss) [reply CONTRIBUTING.md]: Ah, right. Done.

**Review by dixonjoel — APPROVED** [review]

File: examples/nidcpower_source_dc_voltage/README.md [hunk 2]
```diff
@@ -43,18 +42,18 @@ created in NI MAX. To automatically simulate an instrument without using NI MAX,
...
   parent directories (such as the root of your Git repository or
-  `C:\ProgramData\National Instruments\MeasurementLink\Services` for statically
+  `C:\ProgramData\National Instruments\Plug-Ins\Measurements` for statically
   registered measurement services).
 - Add the following options to the `.env` file to enable simulation via the
   driver's option string:
 
   ```
-  MEASUREMENTLINK_NIDCPOWER_SIMULATE=1 
+  MEASUREMENTLINK_NIDCPOWER_SIMULATE=1
```
Comment (bkeryan) [inline examples/nidcpower_source_dc_voltage/README.md:51]: @dixonjoel @DavidCurtiss Outside the scope of this PR: will these also be renamed?

**Review by bkeryan — APPROVED** [review]

File: examples/nidcpower_source_dc_voltage/README.md (hunk 2 above)
Comment (dixonjoel) [reply examples/nidcpower_source_dc_voltage/README.md:51]: I think we should at some point. I'm not too concerned if it makes this release or not. We should probably have an issue for this to remind us at least. I'll file one.

Comment (dixonjoel) [reply examples/nidcpower_source_dc_voltage/README.md:51]: https://github.com/ni/measurement-plugin-python/issues/757

File: README.md [hunk 3]
```diff
@@ -62,39 +62,30 @@ There are three ways to do this:
 
 ## Examples
 
-The `examples` directory contains example measurements for MeasurementLink 2024 Q2 or later. If
-you are using a previous version of MeasurementLink, download the appropriate examples:
-
-- MeasurementLink 2023 Q1: [measurementlink-python-examples-1.0.1.zip](https://github.com/ni/measurementlink-python/releases/download/1.0.1/measurementlink-python-examples-1.0.1.zip)
```
Comment (bkeryan) [inline README.md:68]: @dixonjoel I still think we should point users at the zip file and/or Git tag. I'll post a PR to restore this content.

**Pull request merged**